api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/PaymentCycleEmailHandler.java

```diff
@@ -46,6 +70,76 @@ private void sendChildTurnsFourEmail(PaymentCycle paymentCycle, NextPaymentCycle
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
     }
 
+    private EmailMessagePayload buildChildTurnsFourNotificationEmailPayload(PaymentCycle paymentCycle,
+                                                                            PaymentCycleVoucherEntitlement entitlementNextMonth,
+                                                                            boolean multipleChildrenTurningFourInMonth) {
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle, entitlementNextMonth);
+        emailPersonalisation.put(MULTIPLE_CHILDREN.getTemplateKeyName(), multipleChildrenTurningFourInMonth);
+        return EmailMessagePayload.builder()
+                .claimId(paymentCycle.getClaim().getId())
+                .emailType(EmailType.CHILD_TURNS_FOUR)
+                .emailPersonalisation(emailPersonalisation)
+                .build();
+    }
+
+    private Map<String, Object> createPaymentEmailPersonalisationMap(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement voucherEntitlement) {
+        Claimant claimant = paymentCycle.getClaim().getClaimant();
+        Map<String, Object> emailPersonalisation = new HashMap<>();
+        emailPersonalisation.put(FIRST_NAME.getTemplateKeyName(), claimant.getFirstName());
+        emailPersonalisation.put(LAST_NAME.getTemplateKeyName(), claimant.getLastName());
+        emailPersonalisation.put(PAYMENT_AMOUNT.getTemplateKeyName(), convertPenceToPounds(voucherEntitlement.getTotalVoucherValueInPence()));
+        emailPersonalisation.put(PREGNANCY_PAYMENT.getTemplateKeyName(), buildPregnancyPaymentAmountSummary(voucherEntitlement));
+        emailPersonalisation.put(CHILDREN_UNDER_1_PAYMENT.getTemplateKeyName(), buildUnder1PaymentSummary(voucherEntitlement));
+        emailPersonalisation.put(CHILDREN_UNDER_4_PAYMENT.getTemplateKeyName(), buildUnder4PaymentSummary(voucherEntitlement));
+        emailPersonalisation.put(REGULAR_PAYMENT.getTemplateKeyName(), getRegularPaymentAmount(voucherEntitlement));
+        String formattedCycleEndDate = paymentCycle.getCycleEndDate().plusDays(1).format(DATE_FORMATTER);
+        emailPersonalisation.put(NEXT_PAYMENT_DATE.getTemplateKeyName(), formattedCycleEndDate);
+        return emailPersonalisation;
+    }
+
+    private String getRegularPaymentAmount(PaymentCycleVoucherEntitlement voucherEntitlement) {
+        int totalVouchersForRegularPayment = getRegularPaymentVouchers(voucherEntitlement);
+        return formatRegularPaymentAmount(totalVouchersForRegularPayment, voucherEntitlement.getSingleVoucherValueInPence());
+    }
+
+    private String buildUnder4PaymentSummary(PaymentCycleVoucherEntitlement voucherEntitlement) {
```

Comment:
Might want to add `ForNextCycle` to the end of all these paymentSummary methods.

Reply:
Added ForNextCycle to the method name.